spec/controllers/events_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
       let(:params) { FactoryGirl.attributes_for(:event) }
 
       it 'throws a not authorized error' do
-        expect { subject }.to raise_error(Pundit::NotAuthorizedError)
+        expect(subject).to redirect_to forbidden_path
```

Comment:
As discussed, rescuing the error results in the raise_error check to fail. @brendandeere, do you have any suggestions for this test?